src/Module/Update/Display.php

```diff
@@ -0,0 +1,89 @@
+<?php
+/**
+ * @copyright Copyright (C) 2010-2022, the Friendica project
+ *
+ * @license GNU AGPL version 3 or any later version
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ * See update_profile.php for documentation
+ */
+
+namespace Friendica\Module\Update;
+
+use Friendica\Model\Post;
+use Friendica\Module\Item\Display as DisplayModule;
+use Friendica\Util\DateTimeFormat;
+use Friendica\Network\HTTPException;
+
+/**
+ * Asynchronous update class for the display
+ */
+class Display extends DisplayModule
+{
+	protected function content(array $request = []): string
+	{
+		if ($this->config->get('system', 'block_public') && !$this->session->isAuthenticated()) {
+			throw new HTTPException\UnauthorizedException($this->t('Access denied.'));
+		}
+
+		$profileUid = $request['p'] ?? 0;
+		$force      = $request['force'] ?? false;
+		$uriId      = $request['uri_id'] ?? 0;
```

Comment:
Surprised PHP CS didn't complain about this.
```suggestion
		$profileUid = $request['p']      ?? 0;
		$force      = $request['force']  ?? false;
		$uriId      = $request['uri_id'] ?? 0;
```